mlir/docs/Dialects/emitc.md

```diff
@@ -12,6 +12,10 @@ The following convention is followed:
     operation, C++20 is required.
 *   If `ssize_t` is used, then the code requires the POSIX header `sys/types.h`
     or any of the C++ headers in which the type is defined.
+*   If `_Float16` is used, the code requires the support of C additional
+    floating types.
```

Comment:
I think usually this documentation specifies the type in the mlir world and then relates it to something in C/C++. Maybe
> If `f16` types are used, the code requires a compiler that supports `_Float16`.

Same for `bf16` below.

Reply:
Addressed